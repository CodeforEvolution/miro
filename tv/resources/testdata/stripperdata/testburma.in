<span><img/>
				<p>
				the ongoing burmese military offensive against civilian in Northern Karen state ,eastern Burma
				</p>
				<p>
					Author: <a href="http://youtube.com/profile?user=burmaissuesvideos">burmaissuesvideos</a><br/>
					Keywords:  <a href="/results?search_query=grassroots">grassroots</a> <a href="/results?search_query=outreach">outreach</a><br/>
					Added: April 2, 2008<br/>
				</p></span>
