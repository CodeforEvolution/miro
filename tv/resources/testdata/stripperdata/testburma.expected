(u'the ongoing burmese military offensive against civilian in Northern Karen state ,eastern Burma\nAuthor: burmaissuesvideos\nKeywords: grassroots outreach\nAdded: April 2, 2008', [(103, 120, u'http://youtube.com/profile?user=burmaissuesvideos'), (131, 141, u'/results?search_query=grassroots'), (142, 150, u'/results?search_query=outreach')])
